documentation/source/users/rmg/running.rst

```diff
@@ -20,83 +20,17 @@ Run with CPU profiling::
 
     python rmg.py input.py -p
 
-We recommend you make a job-specific directory for each RMG simulation. Some jobs can take quite a while to complete, so we also recommend using a job scheduler if working in an linux environment. 
-
-The instructions below describe special cases for running an RMG job.
-
-Running RMG in parallel with SLURM
-----------------------------------
-
-RMG has the option to use multiple processes on one node for reaction generation and on-the-fly Quantum Mechanics Thermodynamic Property (QMTP) calculation. Here is an example submission script for an RMG-Py job with a SLURM scheduler.
-
-The job reserves 24 tasks on a single node, but uses only 12 processes in parallel during
-the RMG-Py simulation.
-
-Make sure that: 
-
-- the queue named ``debug`` exists on your SLURM scheduler. 
-- you modify the path to the parent folder of the RMG-Py installation folder.
-- you have an anaconda environment named ``rmg_env`` that contains RMG-Py's dependencies.
-- the working directory from which you launched the job contains the RMG-Py input file ``input.py``
+Run with multiprocessing for reaction generation and QMTP::
 
-.. code:: bash
+    python rmg.py input.py -n <Max number of processes allowed>
```

Comment:
Maybe this is too nitpicky, but I prefer having the optional argument in front of the positional one. I.e., `python rmg.py -n nproc input.py`.

Reply:
Will do